### DISCUSSION TITLE
Discord Rich Presence Idea

### DISCUSSION THREAD

**Discussion by WolfFr1** [pear-devs/pear-desktop#2300] [Ideas]
Hey ive got an idea of a settings to add to discord rpc it would be to directly instead of saying 
Listening to Youtube Music
It would say
Listening: Song Name & Author
![image](https://github.com/user-attachments/assets/0d4779d6-7d77-4090-876f-422f7f4b4992)

**Comment by ArjixWasTaken** [comment]
What you are asking for is impossible.
That is the name of the application, changing it dynamically for each song is not possible.
